.drone.yml

```diff
@@ -16,7 +16,7 @@ pipeline:
       - make lint
     when:
       event: [push]
-      branch: [feature/*, hotfix/*]
+      branch: [*]
```

Comment:
Is this right? Should we not add `bugfix/*` to the list rather?

Reply:
we need to make sure that every build goes thru tests.

I notices that a branch named `issue` didn't went thru the CI test

we can add more branches prefixes

Reply:
maybe remove the branch tag and change event to run on `pull_request`?

```
  test:
    image: kowalatech/go:1.0.1
    commands:
      - make test
      - make lint
    when:
      event: [pull_request]
```

Reply:
Right, I see the problem. `event: [pull_request]` makes the most sense then, since every commit has to go through a PR.